src/test/java/org/embulk/output/elasticsearch/ElasticsearchTestUtils.java

```diff
@@ -46,6 +44,8 @@
     public static String ES_INDEX2;
     public static String ES_ALIAS;
 
+    public static int ES_SUPPORT_TYPELESS_API_VERSION = 7;
```

Comment:
duplicated? 
https://github.com/embulk/embulk-output-elasticsearch/pull/71/files#diff-c29bbe35f7b43a3ff71f5aba04f8a06c7f63c4a3d87c9d589e6d4679bfc48b06R70

Reply:
Rename the variable to avoid confusion